docs/loops.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Loops
+---
+
+_Quick overview: [Loops](overview.md#loops)_
+
+Loops are discouraged in most cases. Instead functional programming patterns
+like `map`, `filter`, or `reduce` can usually be used in their place.
+
+## For Loops
+
+For loops use the `for`, `in`, and `to` keywords. The increment cannot be
```

Comment:
Changed from what? Answer: From 1.
To clarify I think it could say "`for` loops always increment or decrement a counter by one on each iteration".